### PR TITLE
Planning integration adjustments

### PR DESCRIPTION
**Pull request by thecalcc**
STT-108

TODO: 
- [x] Don't show 3 dots menu if no actions
- [x] Don't show toolbars if no actions are there
- [x] Fix print preview & secondary toolbar
- [x] Don't show header collapse button if no fields are in the header
- [x] Shortcuts for print preview, theme switch, theme config work

### CODE REVIEW

**Comment by thecalcc** [comment]
Note - the `SideNav` component has only `className` available as a prop through which you can modify styles. I've went ahead and put a `px-2` class coming from UI framework

**Comment by thecalcc** [comment]
Branch based on `thecalcc:fix-profile-dropdown`

**Comment by thecalcc** [comment]
Header without any fields:

image
image

--- 

Secondary toolbar, that has no actions (doesn't render):
image

--- 

Primary toolbar, no actions (only 3 dots menu):
image

--- 

Primary toolbar, no actions (no 3 dots menu):
image

---

Authoring, no actions on both toolbars (only header actions):
image

---

Authoring, no actions on both toolbars (no header actions):
image

**Review by thecalcc — COMMENTED** [review]
review

File: scripts/apps/ingest/views/dashboard/dashboard.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
     <h2 class="page-nav-title" translate>Ingest Dashboard</h2>
     <div class="button-stack right-stack">
         <div id="ingest-dashboard-dropdown" class="dropdown strict" dropdown sd-tooltip="{{ :: 'Add Sources' | translate }}" flow="left">
-            <button class="navbtn dropdown__toggle sd-create-btn" dropdown__toggle></button>
+            <button class="navbtn dropdown__toggle sd-create-btn" dropdown__toggle>Add sources</button>
```
Comment (tomaskikutis) [inline scripts/apps/ingest/views/dashboard/dashboard.html]: translate string

File: scripts/apps/authoring-react/authoring-react.tsx [hunk 2]
```diff
@@ -1429,15 +1416,13 @@ export class AuthoringReact<T extends IBaseRestApiResponse> extends React.PureCo
...
+                                header={primaryToolbarWidgets.length < 1
+                                    && extraPrimaryToolbarWidgets?.length < 1 ? null : (
                                         <SubNav>
                                             <AuthoringToolbar
                                                 entity={state.itemWithChanges}
                                                 coreWidgets={primaryToolbarWidgets}
-                                                extraWidgets={
-                                                    this.props.getAuthoringPrimaryToolbarWidgets(exposed)
-                                                }
+                                                extraWidgets={extraPrimaryToolbarWidgets}
```
Comment (tomaskikutis) [inline scripts/apps/authoring-react/authoring-react.tsx]: This coreWidgets/extraWidgets thing isn't doing anything, but joins 2 arrays together. Can you simply accept "widgets" as a prop and do the joining of lists before passing?

**Review by tomaskikutis — COMMENTED** [review]
The tests pass here, but there would be z-index issues if we merged this to develop, so let's wait until Nikola fixes it before upgrading develop ui-framework to v4

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**